.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
delete this file

Reply:
done, I'm always struggling with those gitignore things. lol

Reply:
now it looks good